rhtap.groovy

```diff
@@ -0,0 +1,70 @@
+/* Generated from templates/rhtap.groovy.njk. Do not edit directly. */
+
+def info(message) {
+    echo "INFO: ${message}"
+}
+
+def install_script (scriptname) {
+    echo ("Loading libraryResource(${scriptname})")
+    contents = libraryResource( scriptname )
+    writeFile(file:  "rhtap/${scriptname}"  , text: contents)
+    sh "chmod +x rhtap/${scriptname}"
+}
+
+def run_script (scriptname) {
+    // load common utilities across all tasks
+    install_script ("common.sh")
+    install_script ("verify-deps-exist")
+
+    if (scriptname == 'buildah-rhtap.sh') {
+      // Called from buildah-rhtap.sh
+      install_script ('merge-sboms.sh')
+    }
```

Comment:
This part is a little refactor. Compare with https://github.com/redhat-appstudio/tssc-sample-jenkins/blob/154b787c2582dd014f49edbe16fb77554afa8fc5/vars/rhtap.groovy#L21-L23